js-course-m3-TarasYarovyi-home-task

TarasYarovyi opened the pull request:
m2 - printscreen
m2 - [X]
m3 - html

TarasYarovyi committed:
js-course-m3-TarasYarovyi-home-task

m2 - printscreen
m2 - [X]
m3 - html

SerhiiVoznyi commented on M3-JavaScriptBasics/assets/js-course-m3-TarasYarovyi-home-task.html:
```diff
@@ -0,0 +1,17 @@
+<script type="text/javascript"> 
+    alert("This alert box was called with the onload event");
+    function MyFunction() {
+        document.getElementById("home-task").innerHTML = Date();
```
The most valuable ability of any programmer is the ability to search on the internet, 
By writing appropriate requests in Google you will be able to find 50% of all solutions you are interested in. 
For instance, your task is: `current date and time in format DD-MM-YYYY HH:MM:SS` so just by searching it
you will be able to find this: https://stackoverflow.com/questions/10211145/getting-current-date-and-time-in-javascript

TarasYarovyi replied:
Fixed

TarasYarovyi committed:
Update js-course-m3-TarasYarovyi-home-task.html

Date has been changed. The problem was the lack of "new" before Date().
"." was changed to "-"
